sr_robot_launch/launch/sr_hardware_control_loop.launch

```diff
@@ -43,6 +43,8 @@
   <arg name="eth_port" default="$(optenv ETHERCAT_PORT enp2s0)"/>
   <arg name="pwm_control" default="true"/>
   <arg name="hand_trajectory" default="true"/>
+  <arg unless="$(arg arm)" name="initial_y" default="0.0"/>
+  <arg unless="$(arg arm)" name="initial_yaw" default="0.0"/>
```

Comment:
Should this be under the section <!-- ARMS CONFIG--> ?

Reply:
@toliver moved to the arm section in 6d5ce3c